fe/fe-core/src/main/java/com/starrocks/clone/TabletSchedCtx.java

```diff
@@ -1075,73 +1068,6 @@ private String unprotectedFinishClone(TFinishTaskRequest request, Database db, P
                 reportedTablet.getMin_readable_version());
     }
 
-    /*
-     * we try to adjust the priority based on schedule history
-     * 1. If failed counter is larger than FAILED_COUNTER_THRESHOLD, which means this tablet is being scheduled
```

Comment:
> schedule all the tablet in queue one time, because the tablet on the head and tablet on the tail may use different be resource.

I don't understand the reason of this, could you elaborate, if this is the case, what's the point of "priority" queue, will the 
tablet with `VERY_HIGH` priority cannot get scheduled in time?

Reply:
The tablet of high priority in the queue can take the path slot first, the point of priority queue is that the newly added tablet with high priority will be put to the head of queue, and it can be scheduled in time.

Now, the reason of schedule retry is the competition of path slot, we cannot downgrade its priority in this case, the available slot should be allocated to these tablets first.

tablet with `VERY_HIGH` priority can still be scheduled first, because these tablets is stored on the head of queue.